tensornetwork/matrixproductstates/mps.py

```diff
@@ -0,0 +1,644 @@
+# Copyright 2019 The TensorNetwork Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from tensornetwork.network import TensorNetwork
+import numpy as np
+from typing import Any, List, Optional, Text, Type, Union, Dict
+Tensor = Any
+
+
+class FiniteMPS(TensorNetwork):
```

Comment:
I don't think it's a good idea to have FiniteMPS inherit TensorNetwork. I really doubt FiniteMPS should be able to do everything that TensorNetwork can do. I think it would be better to have the network as a property of the FiniteMPS.

Reply:
yes, I think I agree. Wasn't sure about this either, I wanted to try it to see how convenient it is.

Reply:
@Thenerdstation I removed inheritance from `TensorNetwork`